experiment/kerneltest.py

```diff
@@ -79,8 +85,8 @@ def encode_msg(message_type: str, content: dict[str, Any]):
     }
     msg_list: list[bytes] = [
         json.dumps(header).encode(),
-        json.dumps(None).encode(),  # parent_header
-        json.dumps(None).encode(),  # metadata
+        json.dumps({}).encode(),  # parent_header
+        json.dumps({}).encode(),  # metadata
```

Comment:
This script wasn't working until I set this to an empty dict, not sure if it's a Jupyter version issue but seemed like a low-risk change

Reply:
Strange, I wonder what's going on there :D